kv/index_iter_test.go

```diff
@@ -143,3 +143,23 @@ func (s *testIndexSuite) TestIndex(c *C) {
 	err = txn.Commit()
 	c.Assert(err, IsNil)
 }
+
+func (s *testIndexSuite) TestCombineIndexSeek(c *C) {
+	index := kv.NewKVIndex("i", "test", 1, false)
+
+	txn, err := s.s.Begin()
+	c.Assert(err, IsNil)
+
+	values := []interface{}{"abc", "def"}
+	err = index.Create(txn, values, 1)
+	c.Assert(err, IsNil)
+
+	index2 := kv.NewKVIndex("i", "test", 1, false)
+	iter, hit, err := index2.Seek(txn, []interface{}{"abc", nil})
+	defer iter.Close()
```

Comment:
I think it is better to check err first. Panic is not a good thing even if it is in test.